copc/lidar-las-to-copc.ipynb

```diff
@@ -0,0 +1,707 @@
+{
```

Comment:
<p>This would read cleaner to me if you inlined all the arguments:</p><p> </p><p> </p>

<pre class="ql-syntax" spellcheck="false">las_item_results = earthaccess.search_data(
&nbsp;&nbsp;&nbsp; short_name="GLLIDARPC",
&nbsp;&nbsp;&nbsp; version="001",
&nbsp;&nbsp;&nbsp; temporal = ("2020"),
&nbsp;&nbsp;&nbsp; count=3
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I agree. I've updated the same in the revision.</p>